src/lib.rs

```diff
@@ -169,9 +169,9 @@ impl AutoCfg {
         let mut ac = AutoCfg {
             rustflags: rustflags(&target, &dir),
             out_dir: dir,
-            rustc: rustc,
-            rustc_version: rustc_version,
-            target: target,
+            rustc,
+            rustc_version,
+            target,
```

Comment:
This crate aims to keep extreme compatability, all the way to Rust 1.0, but this form of field initialization requires 1.17.